v0/destinations/google_ads/config.js

```diff
@@ -0,0 +1,15 @@
+const { getMappingConfig } = require("../../util");
+
+const baseEndpoint = "hhttps://www.googleadservices.com/pagead/conversion/app/";
+
+const CONFIG_CATEGORIES = {
+    TRACK: {  name: "GoogleAdsTrackConfig" }
+  };
+  
+  const MAPPING_CONFIG = getMappingConfig(CONFIG_CATEGORIES, __dirname);
+  
+  module.exports = {
+    CONFIG_CATEGORIES,
+    MAPPING_CONFIG,
```

Comment:
As we have a single type of event, we should move this extra structurization. Something like 
```
trackConfig: MAPPING_CONFIG[CONFIG_CATEGORIES.TRACK]
```
Then in `transform.js` we don't need to pass the extra param: `category` as it's already de-structured here.

Reply:
done